source/_data/templates.yml

```diff
@@ -0,0 +1,6 @@
+- name: Embark Vortex Template
+  description: A React Application showing what can be done with Vortex when using Embark.
+  link: https://github.com/Horyus/vortex-demo-embark/
```

Comment:
Maybe we could add the cmd to use the template eg: `embark new --template=link-to-template`
Unless I misunderstood how this is supposed to be used